tools/webdriver/webdriver/client.py

```diff
@@ -425,7 +429,26 @@ def send_command(self, method, url, body=None):
             an error.
         """
         response = self.transport.send(method, url, body)
-        value = response.body["value"]
+
+        if "value" in response.body:
+            value = response.body["value"]
+        elif "error" in response.body:
+            logger.warning("'value' key not found in response but 'error' key "
+                           "found in root. This is a compliance issue, but the tests are "
+                           "working around it.")
+            value = response.body
```

Comment:
Because if we fail to start a session due to non-conformance of the new-session response we can't run any tests. I don't see why we would make getting that specific detail right a prerequisite to determining interop in other areas. To be clear we should catch this error with a specific test.